src/Altinn.App.PlatformServices/Implementation/AppBase.cs

```diff
@@ -43,17 +35,13 @@ public abstract class AppBase : IAltinnApp
         private readonly IAppResources _resourceService;
         private readonly IProcess _processService;
```

Comment:
If this is really unused should we remove it?

Reply:
Fortgot about these, yes I'll remove them.